tests/src/util/docker.ts

```diff
@@ -56,13 +56,13 @@ export class Docker implements DropHandler {
     return port
   }
 
-  async runPebble(): Promise<PebbleResult> {
+  async runPebble(eab = false): Promise<PebbleResult> {
     const port = await getPort()
     const tempdir = mkdtempSync(join(tmpdir(), "spawner-pebble-config-"))
     const certs = await generateCertificates()
     const imageName = "docker.io/letsencrypt/pebble:latest"
 
-    const pebbleConfig = {
+    const pebbleConfig = eab ? {
```

Comment:
rather than do this (which creates the possibility of accidental drift between the duplicated fields, like `httpPort`), I'd rather have one `pebbleConfig` definition that [spread syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) in the eab configuration. For example:

```
const eabConfig = {};
if (eab) {
    eabConfig = { 
        externalAccountBindingRequired: true,
        // etc.
    }
}
const pebbleConfig = {
    pebble: {
        listenAddress: "blah",
        // etc.
        ...eabConfig
    }
}
```